main.js

```diff
@@ -1,12 +1,123 @@
+// class weekSchedule {
+//   constructor{
+// this.day = {}
+//   }
+// }
+
+var schedule = {
+  Sunday: {
+    day: 'Sunday',
+    isSelected: false,
+    todo: {
+    }
+  },
+  Monday: {
+    day: 'Monday',
+    isSelected: false,
+    todo: {
+    }
+
+
+  },
+  Tuesday: {
+    day: 'Tuesday',
+    isSelected: false,
+    todo: {
+    }
+
+
+  },
+  Wednesday: {
+    day: 'Wednesday',
+    isSelected: false,
+    todo: {
+    }
+
+
+  },
+  Thursday: {
+    day: 'Thursday',
+    isSelected: false,
+    todo: {
+    }
+
+
+  },
+  Friday: {
+    day: 'Friday',
+    isSelected: false,
+    todo: {
+    }
+
+
+  },
+  Saturday: {
+    day: 'Saturday',
+    isSelected: false,
+    todo: {
+    }
+
+
+  }
+};
+
+
+
 var days = document.querySelector('.days');
-var h2 = document.querySelector('h2');
-var selectedDay = '';
+var modal = document.querySelector('.modal');
+var addEntryBtn = document.querySelector('.add-entry');
+var form = document.querySelector('form');
+
 
-days.addEventListener('click', function (event) {
-  if (event.target===days){
-    return
+var scheduleDay = document.querySelector('#day');
+var scheduleTime = document.querySelector('#time');
+var textarea = document.querySelector('textarea');
+
+
+days.addEventListener('click', selectedDay);
+addEntryBtn.addEventListener('click', showModal);
+form.addEventListener('change', setSchedule);
```

Comment:
You want to listen for a submit event, it shouldnt update everytime the user changes something , what if they press submit with the default values? then there would be nothing changed and nothing added